src/test/resources/ays_ui_elements_en.properties

```
@@ -0,0 +1,3 @@
+emailAddress_errorMessage=Invalid E-mail.
+password_errorMessage=This field must be at least 6 characters.
+header=Welcome
```

`.properties` uzantılı dosyalarda standart olabilmesi adına kelimeler `.` karakteri ile ayırılır. Ek olarak uzun vadeli düşündüğümüz durumda aşağıdaki dosya da şu şekilde yazılabilir. Buranın standartları ile ilgili de bir doküman hazırlanabilir. `<module>.<category>.<key>=<value>`

```suggestion
login.validation_error.email_address=Invalid E-mail.
login.validation_error.password=This field must be at least 6 characters.
login.header.welcome=Welcome

```